spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -11,6 +11,192 @@ We then build an introductory Spring Cloud Data Flow application, discussing som
 
 
 
+[[getting-started-deploying-spring-cloud-dataflow-quick-start]]
```

Comment:
I'd suggest moving this section after "System Requirements". 

Also, for consistency, we could rename the section headers:
1) "Quick Start using Docker Compose" -> "Getting Started with Docker Compose"
2) "Installation" -> "Getting Started with Manual Installation"

In summary:
- Getting started
  - System Requirements
  - Getting Started with Docker Compose
  - Getting Started with Manual Installation
  - .... 
  - ...